react-app/src/pages/ErrorFallback.tsx

```diff
@@ -158,6 +158,7 @@ const ErrorFallback = ({ error, resetErrorBoundary }) => {
                       },
                       userMessage: text,
                       timestamp: new Date().toLocaleString(),
+                      url: window.location.href,
                     })
                     .then((res) => {
                       if (res.status === 200) {
```

Comment:
Could be good to make this a loading button and handle some loading state so that you can't press the submit button again while the request is going through. As is I'm able to send multiple emails if I spam the button. Could be handled either with useDataSubmitter or just its own state hook here.

Reply:
Updated to use LoadingButton component.